.github/workflows/autorelease-tag.yml

```diff
@@ -19,6 +19,9 @@ jobs:
         id: get_commit
         run: git rev-list  `git rev-list --tags --no-walk --max-count=1`..HEAD --count | xargs -I {} echo COMMIT_COUNT={} >> $GITHUB_OUTPUT
 
+      - name: Bump version in banner.go
+        run: bash scripts/bump_version.sh internal/runner/banner.go patch
+
```

Comment:
@dogancanbakir , have you tried this workflow ??  i think (if i am not wrong) this will not be reflected in actual release binary because we did not commit any changes !!